src/main/java/com/google/api/codegen/CodeGeneratorApi.java

```diff
@@ -61,9 +59,14 @@
           "The list of YAML configuration files for the code generator.",
           ImmutableList.<String>of());
 
-  /**
-   * Constructs a code generator api based on given options.
-   */
+  public static final Option<Boolean> DISABLE_TEST_GEN =
```

Comment:
I feel filter_artifacts can be ambiguous since I don't know if the listed artifacts are going to be enabled or disabled. I picked `enabled_artifacts` instead.